recipe/meta.yaml

```
@@ -1,42 +1,38 @@
-{% set version = "9.0.0" %}
+{% set version = "10.0.0" %}
 {% set cuda_enabled = cuda_compiler_version != "None" %}
 {% set build_ext_version = "3.0.0" %}
```

You should also change this as we are changing how the `-proc` packages work. Otherwise, we could also get rid of `build_ext_version` and always use `version` instead.
```suggestion
{% set build_ext_version = "10.0.0" %}
```

---

It's a fair point; though I'm actually taking care to ensure it keeps working as before. Don't mind bumping the version though.